This file is part of MXE.
See index.html for further information.

From 9b8bd267588f9c7e15b50bf6f7754837deb8b301 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Tue, 3 Apr 2012 13:58:42 +0200
Subject: [PATCH 1/5] Revert "make dist will also make lzip compressed tarball"

This reverts commit 97ac1bc58274f651338444693bd75441ba12f46f.

diff --git a/configure.ac b/configure.ac
index 3e5b79d..99f3636 100644
--- a/configure.ac
+++ b/configure.ac
@@ -26,7 +26,7 @@ AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CANONICAL_HOST
 
-AM_INIT_AUTOMAKE([1.12.2 subdir-objects no-dist-gzip dist-xz dist-lzip -Wall -Wno-override])
+AM_INIT_AUTOMAKE([1.12.2 subdir-objects no-dist-gzip dist-xz -Wall -Wno-override])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 AC_CONFIG_HEADERS([config.h])
 
-- 
2.1.2


From da0afaf172601c7bc434f35a4d02c13811fcaee7 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Wed, 6 Jun 2012 09:57:24 +0200
Subject: [PATCH 2/5] AM_PROG_AR for automake 1.12 compatibility

Taken from
http://lists.gnu.org/archive/html/automake/2012-05/msg00014.html

diff --git a/configure.ac b/configure.ac
index 99f3636..8c1b4fd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -42,6 +42,7 @@ ggl_EARLY
 AM_PROG_AS
 AM_PROG_AR
 AC_PROG_CXX
+m4_ifdef([AM_PROG_AR], [AM_PROG_AR])
 AM_PROG_CC_C_O
 AC_PROG_YACC
 AC_PROG_SED
-- 
2.1.2


From 4a98dd8e1a9c6d2d1f1e190f94582ab259435c84 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Mon, 4 Feb 2013 16:11:12 +0100
Subject: [PATCH 3/5] add missing private lib to pc file


diff --git a/lib/gnutls.pc.in b/lib/gnutls.pc.in
index a25fcdf..3860c68 100644
--- a/lib/gnutls.pc.in
+++ b/lib/gnutls.pc.in
@@ -19,6 +19,6 @@ Description: Transport Security Layer implementation for the GNU system
 URL: http://www.gnutls.org/
 Version: @VERSION@
 Libs: -L${libdir} -lgnutls
-Libs.private: @LTLIBZ@ @LTLIBINTL@ @LIBSOCKET@ @LTLIBPTHREAD@ @LTLIBICONV@ @P11_KIT_LIBS@ @LIB_SELECT@ @TSS_LIBS@ @GMP_LIBS@
+Libs.private: @LTLIBZ@ @LTLIBINTL@ @LIBSOCKET@ @LTLIBPTHREAD@ @LTLIBICONV@ @P11_KIT_LIBS@ @LIB_SELECT@ @TSS_LIBS@ @GMP_LIBS@ -lcrypt32
 @GNUTLS_REQUIRES_PRIVATE@
 Cflags: -I${includedir}
-- 
2.1.2


From 57ffca60a0d61c4b58e27f6ee7f9b2770b0011b6 Mon Sep 17 00:00:00 2001
From: Tony Theodore <tonyt@logyst.com>
Date: Wed, 6 Feb 2013 08:43:43 +0100
Subject: [PATCH 4/5] relax automake version check


diff --git a/configure.ac b/configure.ac
index 8c1b4fd..efa13e8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -26,7 +26,7 @@ AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 AC_CANONICAL_HOST
 
-AM_INIT_AUTOMAKE([1.12.2 subdir-objects no-dist-gzip dist-xz -Wall -Wno-override])
+AM_INIT_AUTOMAKE([1.11.1 subdir-objects no-dist-gzip dist-xz -Wall -Wno-override])
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES([yes])])
 AC_CONFIG_HEADERS([config.h])
 
-- 
2.1.2


From e6872f0073970fabe4cc267feebb8bad98756f72 Mon Sep 17 00:00:00 2001
From: Mark Brand <mabrand@mabrand.nl>
Date: Mon, 24 Nov 2014 08:56:48 +0100
Subject: [PATCH 5/5] windows build fix: ws2tcpip.h supplies inet_ntop

Follow-up to 492c2b937ab66134d0b37499a6f3a747e19bc31a

Signed-off-by: Mark Brand <mabrand@mabrand.nl>

taken from: http://lists.gnutls.org/pipermail/gnutls-devel/2014-November/007250.html

diff --git a/lib/x509/output.c b/lib/x509/output.c
index bf01834..1ec18de 100644
--- a/lib/x509/output.c
+++ b/lib/x509/output.c
@@ -32,7 +32,11 @@
 #include <extras/randomart.h>
 
 #ifdef HAVE_INET_NTOP
-# include <arpa/inet.h>
+# ifdef _WIN32
+#  include <ws2tcpip.h>
+# else
+#  include <arpa/inet.h>
+# endif
 #endif
 
 #define addf _gnutls_buffer_append_printf
-- 
2.1.2

